src/metatrain/utils/data/dataset.py

```diff
@@ -77,12 +85,19 @@ class DatasetInfo:
 
     def __init__(
         self,
-        length_unit: Optional[str],
+        length_unit: str,
         atomic_types: List[int],
         targets: Dict[str, TargetInfo],
         extra_data: Optional[Dict[str, TargetInfo]] = None,
     ):
-        self.length_unit = length_unit if length_unit is not None else ""
+        # verify metadata
+        _ = ModelCapabilities(
+            outputs={"energy": ModelOutput()},
+            length_unit=length_unit,
+            atomic_types=atomic_types,
+        )
```

Comment:
What is this doing?

Reply:
in metatomic we check if the `length_unit` and `atomic_types` are valid. 

I think the is good to do here otherwise users only will get an error when the model is be exported (after training). This might be frustrating if there is a typo in the unit.